backend/routers/social_media_marketing_router.py

```
@@ -828,56 +849,103 @@ async def get_swamiji_avatar_config(request: Request, admin_user: dict = Depends
         ) from e
 
 
-@social_marketing_router.post("/generate-avatar-preview")
+# CORE.MD: Use a module-level constant for better maintainability.
+AVAILABLE_AVATAR_STYLES = ["traditional", "modern", "default"]
+
+@router.post("/generate-avatar-preview", response_model=StandardResponse, tags=["Social Media Marketing", "Admin"])
```

Ruff (0.12.2)

Undefined name `router`

(F821)